packages/grafana-prometheus/jest.config.js

```diff
@@ -0,0 +1,5 @@
+const sharedConfig = require('../../jest.config.js');
+module.exports = {
+  ...sharedConfig,
+  rootDir: '../../',
+};
```

Comment:
This is not related to the PR but necessary for testing with Intellij IDEs.
![image](https://github.com/grafana/grafana/assets/820946/90b6a4d0-aebe-417e-a23a-80026b30c49f)